### PR TITLE
put manage students flow all in separate routes

### PR DESCRIPTION
**Pull request by NilsBacke**

### CODE REVIEW

**Comment by NilsBacke** [comment]
[Move Single Student View and Expanded Student View to separate routes with id's in params](https://trello.com/c/mXlq0PwH/270-move-single-student-view-and-expanded-student-view-to-separate-routes-with-ids-in-params)

File: frontend/src/advising/ManageStudents/StudentView.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+      setFetchingStudent(false);
+    });
+    findAllPlansForUser(id).then((plans: IPlanData[]) => {
+      dispatch(setUserPlansAction(plans, 2020));
+      if (!plans || !plans.length || !plans[0].schedule) setNoPlans(true);
+    });
+  }, []);
+
+  return (
+    <Container>
```
Comment (ArunJ98) [inline frontend/src/advising/ManageStudents/StudentView.tsx:145]: nit but can we break up this JSX? this render function is pretty big, could we split it into smaller components? For example, we could have the StudentPreviewContainer and SchedulePreviewContainer as different components

File: frontend/src/services/AdvisorService.ts [hunk 2]
```diff
@@ -1,5 +1,18 @@
 import { getAuthToken } from "../utils/auth-helpers";
 
+export interface IAbrStudent {
+  fullName: string;
```
Comment (ArunJ98) [inline frontend/src/services/AdvisorService.ts:4]: I added a IAbrUser that's very similar to this to the types folder. Can we merge the two?

**Comment by ArunJ98** [comment]
Overall looks really good, just some minor comments

File: frontend/src/services/AdvisorService.ts (hunk 2 above)
Comment (NilsBacke) [reply frontend/src/services/AdvisorService.ts:4]: Seems like they're for two different use cases, so I renamed the existing one to `IAbrAdvisor`

**Review by ArunJ98 — APPROVED** [review]

**Pull request merged**